examples/circle.ml

```diff
@@ -1,12 +1,15 @@
-open Graphics
+open Joy.Shape
 
 let () =
-  open_graph " 300x300";
-  (* Open a graphics window with dimensions 800x600 *)
-  set_color black;
+  init ();
 
-  draw_circle 150 150 50;
+  (* Create a black circle *)
+  let black_circle = circle 50 in
 
-  ignore (read_line ());
-  close_graph ();
-  exit 0
+  (* Create a colored circle with a yellow fill and a red stroke *)
+  let colored_circle = circle ~fill:(255, 255, 0) ~stroke:(255, 0, 0) 50 in
+
+  (* Show both circles *)
+  show [black_circle; colored_circle];
+
+  close ()
```

Comment:
Can you create a new example for your changes please, rather than over-writing an old one?